### PR TITLE
Fix the Failed to load pdf file error of postman

### PR DESCRIPTION
**Pull request by YoginiTayade**

### CODE REVIEW

File: package-lock.json
Comment (snehal0904) [inline package-lock.json:1]: @YoginiTayade Remove this file add in git ignore

File: src/sso/sso.services.ts [hunk 1]
```diff
@@ -3623,13 +3622,17 @@ export class SSOService {
...
     let qrcodestring = await qr.toDataURL(stringData, function (err, code) {
       if (code) {
         //return console.log('error');
 
         //console.log(code);
         qrcodestring = code;
+        console.log(code);
         return qrcodestring;
+      } else {
+        console.log(err, '------------------');
```
Comment (snehal0904) [inline src/sso/sso.services.ts:3635]: @YoginiTayade Remove console or add proper error message

File: src/sso/sso.services.ts [hunk 2]
```diff
@@ -3623,13 +3622,17 @@ export class SSOService {
...
 
     let stringData = JSON.stringify(url);
+
     let qrcodestring = await qr.toDataURL(stringData, function (err, code) {
       if (code) {
         //return console.log('error');
 
         //console.log(code);
         qrcodestring = code;
+        console.log(code);
```
Comment (snehal0904) [inline src/sso/sso.services.ts:3632]: @YoginiTayade Remove console

File: src/sso/sso.services.ts [hunk 3]
```diff
@@ -344,6 +348,7 @@ export class SSOService {
           const response = await promise;
           //console.log(JSON.stringify(response.data));
           render_response = response.data;
+          //console.log(render_response);
```
Comment (snehal0904) [inline src/sso/sso.services.ts:351]: @YoginiTayade Remove commented code

**Review by snehal0904 — CHANGES_REQUESTED** [review]

File: src/sso/sso.services.ts (hunk 3 above)
Comment (YoginiTayade) [reply src/sso/sso.services.ts:351]: Resolved

File: src/sso/sso.services.ts (hunk 2 above)
Comment (YoginiTayade) [reply src/sso/sso.services.ts:3632]: Resolved

File: src/sso/sso.services.ts (hunk 1 above)
Comment (YoginiTayade) [reply src/sso/sso.services.ts:3635]: Resolved

**Pull request merged**